bpf/init.sh

```diff
@@ -14,30 +14,32 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-LIB=$1
-RUNDIR=$2
-IP4_HOST=$3
-IP6_HOST=$4
-MODE=$5
+LIB=${1}
+RUNDIR=${2}
+PROCSYSNETDIR=${3}
+SYSDIR=${4}
```

Comment:
nit: Since we're only using `${SYSDIR}/class/net`, the "asymmetry" with `${PROCSYSNETDIR}` looks odd to me. In other words, either one of:

* `${PROCDIR}` for `/proc` and `${SYSDIR}` for `/sys`
* `${PROCSYSNETDIR}` for `/proc/sys/net` and `${SYSCLASSNETDIR}` for `/sys/class/net`

would "feel cleaner" to me, does that make sense?